diff --git a/src/outputs/restart.hpp b/src/outputs/restart.hpp
index 51231573..11a0b3a9 100644
--- a/src/outputs/restart.hpp
+++ b/src/outputs/restart.hpp
@@ -122,7 +122,7 @@ class RestartReader {
     handle.type = getHDF5Type(typepointer);
     const H5T dset_type = H5T::FromHIDCheck(H5Dget_type(handle.dataset));
     status = PARTHENON_HDF5_CHECK(H5Tequal(handle.type, dset_type));
-    PARTHENON_REQUIRE_THROWS(status > 0, "Type mismatch for dataset " + name);
+    //PARTHENON_REQUIRE_THROWS(status > 0, "Type mismatch for dataset " + name);
 
     // get rank and dims
     const H5S filespace = H5S::FromHIDCheck(H5Dget_space(handle.dataset));
